docs/faq/admin.txt

```diff
@@ -116,5 +116,5 @@ What assistive technologies are supported for using the admin?
 The admin is intended to be compatible with a wide range of assistive
 technologies, but there are currently many blockers. The support target is all
 latest versions of major assistive technologies, including Dragon, JAWS, NVDA,
-Orca, TalkBack, Voice Control, VoiceOver, Windows Contrast Themes, ZoomText,
-and screen magnifiers.
+Orca, TalkBack, Voice Control, VoiceOver iOS, VoiceOver macOS,
+Windows Contrast Themes, ZoomText, and screen magnifiers.
```

Comment:
Wrap at 79 chars.
```suggestion
Orca, TalkBack, Voice Control, VoiceOver iOS, VoiceOver macOS, Windows Contrast
Themes, ZoomText, and screen magnifiers.
```

Reply:
Could you explain (or point me at a reference) why it has to be exactly at 79 characters? I can kind of understand the point in wrapping, but here doing it at 79 just makes the text harder to read

Reply:
I'm not sure what I need to explain :upside_down_face: We wrap docs, comments, and docstrings at 79 chars. It's documented as wrap at 79 chars, not _"wrap at 79 chars but if you think that something else is more readable feel-free to wrap in a different way"_ :wink:  .

Reply:
I’d like you to explain _why_ we do that, or I suppose why it has to be exactly at 79 characters. Because here the content is clearly easier to work with if we could wrap earlier.

"Contrast Themes" is the name of the feature, it’s not two random words in a sentence, so it’s more readable if the words are side by side. And it’s much easier to find references in the docs if we don’t split nouns like that in two lines.

Reply:
> I’d like you to explain why it has to be exactly at 79 characters, aside from "the docs say so".

The reasons behind this have never been important to me, and I'm really surprised that it's so important to you. That's how I've worked since my first contribution. Do you really want to die on this hill :shrug:? Sorry, but I'm not going to investigate the reasons. The project style guide should be accepted by contributors. If we discussed why 79, why 88, why double quotations etc., every time, we wouldn't do anything else, TBH.

Reply:
Hmm sorry, didn’t mean to be a pain here. I see myself writing more documentation for Django in the future, so I’d rather make sure I understand the style guide where there is room for interpretation. At least to me it very much reads as "wrap at 79 chars but if you think that something else is more readable feel-free to wrap in a different way".

> Wrap the documentation at 80 characters wide, unless a code example is significantly less readable when split over two lines, or for another good reason.
> [Guidelines for reStructuredText files](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files)